Makefile

```diff
@@ -27,6 +27,15 @@ local:
 	mkdir -p bin
 	go build -ldflags "-X main.vendorVersion=${VERSION}" -o bin/gcfs-csi-driver ./cmd/
 
+windows: windows-local
+	cp Dockerfile Linux-Dockerfile
+	cp test/experimental/Dockerfile Dockerfile
+	docker build --build-arg TAG=$(VERSION) -t $(IMAGE)-windows:$(VERSION) .
```

Comment:
We can use "-f" option to pass in a non-default Dockerfile.  That way we don't need to mess with the normal one.

Reply:
Done.